src/librustdoc/formats/mod.rs

```diff
@@ -43,8 +44,7 @@ impl Impl {
     // Returns true if this is an implementation on a "local" type, meaning:
     // the type is in the current crate, or the type and the trait are both
     // re-exported by the current crate.
-    pub(crate) fn is_on_local_type(&self, cx: &Context<'_>) -> bool {
-        let cache = cx.cache();
+    pub(crate) fn is_on_local_type(&self, cache: &Cache, tcx: TyCtxt<'_>) -> bool {
```

Comment:
Why make this change? All the call-sites seem to have a `Context`.

Reply:
At first I tried to use this function in the JSON logic, so I had to make it output-format-agnostic, but after I got rid of it, I realised keeping it that way might be beneficial in the future

Reply:
Could you revert this?

Reply:
Would you accept such a change in a separate PR? possibly expanded if I find other cases of output-format-agnostic functionality relying on a specific output format

Reply:
I don't think so. As long as this method's only used in the HTML backend, I (personally) think it's fine to rely on HTML-specific helpers.